### PR TITLE
epic5: init at 2.0.1

### PR DESCRIPTION
**Pull request by ndowens**
###### Motivation for this change

Add Epic5 to nixpkg repo

###### Things done

- [ x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x ] NixOS
   - [ ] macOS
   - [ ] Linux
- [x ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x ] Tested execution of all binary files (usually in `./result/bin/`)
- [x ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by ndowens** [comment]
I am currently trying to fix the 2nd linux build, in which i think is either ubuntu or debian;

**Comment by ndowens** [comment]
@LnL7

File: pkgs/applications/networking/irc/epic5/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  src = fetchurl {
+    url = "http://ftp.epicsol.org/pub/epic/EPIC5-PRODUCTION/${name}.tar.xz";
+    sha256 = "1ap73d5f4vccxjaaq249zh981z85106vvqmxfm4plvy76b40y9jm";
+    };
+
+   # Darwin needs libiconv, tcl; while Linux build don't
+ 
+   buildInputs = [ openssl ncurses ]  
+   ++ stdenv.lib.optional
```
Comment (LnL7) [inline pkgs/applications/networking/irc/epic5/default.nix]: While it works, it's preferred not to have nested lists in buildInputs. You can use `stdenv.lib.optionals` to add a list of conditional inputs.

Comment (ndowens) [inline pkgs/applications/networking/irc/epic5/default.nix]: Done

**Comment by LnL7** [comment]
I'm not sure why the binary segfaults on darwin. I did find these configure flags when looking at the homebrew definition, perhaps you should include them. https://gist.github.com/LnL7/705c5800e52be9a0fac397b90a34893a#file-epic5-patch-L36

**Review by LnL7 — APPROVED** [review]

**Pull request merged**